tasks/collect-mrrc-params/collect-mrrc-params.yaml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: collect-mrrc-params
+spec:
+  params:
+    - name: dataJsonPath
+      type: string
+      description: path to data json file
+  results:
+    - name: mrrcParamFilePath
+      description: path of the env file for following tasks
+    - name: cosignPubKeyConfig
+      description: the configmap name for cosign verify public key
+    - name: charonAWSSecret
+      description: the secret name for charon aws credential file
+  steps:
+    - name: collect-mrrc-params
+      image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+      script: |
+        set -eux
+
+        MRRC_ENV_FILE_PATH="$(dirname $(params.dataJsonPath))/mrrc.env"
+
+        environment="$(jq -jer '.mrrc.environment' $(params.dataJsonPath))"
```

Comment:
It's copied from the collect-pyxis-params task: https://github.com/konflux-ci/release-service-catalog/blob/development/tasks/collect-pyxis-params/collect-pyxis-params.yaml#L49

Seems the -r will bring a newline, actually I don't think a newline is needed here, so maybe -j is needed other than -r. I'll fix with -j instead of -r

Reply:
Fixed.

Reply:
> It's copied from the collect-pyxis-params task

I wasn't aware it was used there. But yeah, that seems wrong :)

I would preferably go with `-r` as that's what we use in most places and should be sufficient (the newline doesn't matter - it will get stripped before it's assigned to the variable). But `-j` seems to be ok too.